### PR TITLE
chore: e2e report

### PR DESCRIPTION
**Pull request by BCerki**

### CODE REVIEW

File: .github/workflows/test.yaml
Comment (shon-button) [inline .github/workflows/test.yaml:1]: line 383: change path to ensure the failure reports are successfully uploaded.

FROM:
`path: bciers/apps/registration/blob-report`
TO:
`path: bciers/blob-report`

Comment (shon-button) [inline .github/workflows/test.yaml:1]: line 342 np:

`- name: 🎭 Run Playwright Tests`

why let an amazing icon got to waste ? 🤔

Comment (shon-button) [inline .github/workflows/test.yaml:1]: after line 400 `run: npx playwright merge-reports --reporter html ./all-blob-reports`: missing section to upload the merged report

```
      - name: Upload HTML report
        uses: actions/upload-artifact@v4
        with:
          name: playwright-report
          path: playwright-report
          retention-days: 14
  # Ensure the e2e tests and e2e report completed successfully
```

Comment (BCerki) [reply .github/workflows/test.yaml:1]: worst mistake of my career

**Review by shon-button — APPROVED** [review]
💫

**Pull request merged**